CHANGELOG.md

```diff
@@ -1,5 +1,98 @@
+# v0.25.0 Release - 3/15/2019
+
+*Note*: This release comes with a config change, use `skaffold fix` to permanently upgrade your config to `v1beta7`, however old versions are now auto-upgraded. 
+See [deprecation-policy.md](/deprecation-policy.md) for details on what beta means.
+
+
+*Deprecation notice*: With this release we mark for deprecation the `additionalFlags` field in kaniko, instead the kaniko builder plugin .
+This flag will will be removed earliest 06/15/2019. 
+
+New features: 
+
+* Config upgrade: handle helm overrides [#1646](https://github.com/GoogleContainerTools/skaffold/pull/1646)
+* make initContainer in localDir type of kaniko build param [#1727](https://github.com/GoogleContainerTools/skaffold/pull/1727)
```

Comment:
umm, maybe not an expert, so i cant understand what this means. However, digging into the PR, the description makes more sense.
shd we change this to
"Add init container image used to run init container which mounts kaniko context."